CHANGELOG.md

```diff
@@ -4,10 +4,38 @@
 
 ### Features
 
-- **auth**: Fix cancelling hostedUI returning a generic error (#982)
 - Support lazy load and pagination for API (#1009)
 - Separate DataStore List logic out to list provider (#1000)
 
+### Behavior Change
+
+- **Auth**: Fix cancelling hostedUI returning a generic error (#982).
+When a user cancels the sign in flow from `Amplify.Auth.signInWithWebUI`, the callback will now deliver an error of type `AuthError.service`, with an underlying error of `AWSCognitoAuthError.userCancelled`. Previously, the error was a `SFAuthenticationError.canceledLogin`. The `AuthError.service` error type will be returned regardless of whether the underlying HostedUI operation was performed via an SFAuthenticationSession or an ASWebAuthenticationSession.
+    
+You can detect the user cancellation case by `switch`ing on the error, as in:
+    
+```swift
+switch result {
+  case .success:
+      print("SignOut")
+  case .failure(let error):
+      if case AuthError.service(_, _, let underlyingError) = error,
+          case .userCancelled = (underlyingError as? AWSCognitoAuthError) {
+          print("User cancelled")
+      }
+}
+```
```

Comment:
nit: add blank line above L13. Indent from 13-27 by 4 spaces so they'll be indented under bullet.